src/App.jsx

```diff
@@ -1,21 +1,32 @@
 import { useState, useEffect } from 'react'
 import './App.css'
+import {Routes, Route} from "react-router-dom"
 import Header from './components/Header'
 import Articles from './components/Articles'
+import Article from './components/Article'
 import Searchbar from './components/Searchbar'
-import {Routes, Route} from "react-router-dom"
 
 
 function App() {
-
   const [articles, setArticles] = useState([])
 
   return (
+    <>
     <div className="HomeArticles">
-      <Header />
-      <Searchbar />
-      <Articles articles={articles} setArticles={setArticles}/>
+    <Header />
+    <Searchbar />
+      <Routes>
+        <Route path = "/" element={
+          <div>
+            <Articles articles={articles} setArticles={setArticles}/>
+          </div>
```

Comment:
There is no need for the element here to be a div containing a component, the element shown on a path should just be a single component. I also don't see why you have state in App here to be used in Articles, what is the benefit??